virtio-devices/Cargo.toml

```diff
@@ -25,6 +25,7 @@ seccomp = { git = "https://github.com/firecracker-microvm/firecracker", tag = "v
 serde = ">=1.0.27"
 serde_derive = ">=1.0.27"
 serde_json = ">=1.0.9"
+timerfd = ">=1.0"
```

Comment:
Could you please try to rely on the `TimerFd` implementation coming from `vmm-sys-util`?

Reply:
The PR now is using `vmm-sys-util::TimerFd`. I kept a split commit that contains only the changes related to `TimerFd`. You may find it easier to review (without reading all the rate_limiter.rs implantation: https://github.com/likebreath/cloud-hypervisor/commit/07c4ea302a0eb3213eadd70b031468e26fbb03b8.